trieste/bayesian_optimizer.py

```diff
@@ -99,6 +129,19 @@ def try_get_final_models(self) -> Mapping[str, TrainableProbabilisticModel]:
         """
         return self.final_result.unwrap().models
 
+    def try_get_final_model(self) -> TrainableProbabilisticModel:
+        """
+        Convenience method to attempt to get the final model for a single-objective run.
+
+        :return: The final model, if the optimization completed successfully.
```

Comment:
A multi-objective run could still have only a single model? (e..g with the modelstack) Do you mean a single-model run?